src/catalog/src/builtin.rs

```diff
@@ -2520,39 +2530,40 @@ cluster_name, transaction_isolation, execution_timestamp, transaction_id,
 transient_index_id, began_at, finished_at, finished_status,
 error_message, rows_returned, execution_strategy
 FROM mz_internal.mz_statement_execution_history",
-    sensitivity: DataSensitivity::SuperuserAndSupport,
-};
+    access: vec![SUPPORT_SELECT, MONITOR_REDACTED, MONITOR],
+});
 
 pub static MZ_PREPARED_STATEMENT_HISTORY: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
     name: "mz_prepared_statement_history",
     schema: MZ_INTERNAL_SCHEMA,
     data_source: Some(IntrospectionType::PreparedStatementHistory),
     desc: MZ_PREPARED_STATEMENT_HISTORY_DESC.clone(),
     is_retained_metrics_object: false,
-    sensitivity: DataSensitivity::Superuser,
+    access: vec![MONITOR],
 });
 
-pub static MZ_PREPARED_STATEMENT_HISTORY_REDACTED: BuiltinView = BuiltinView {
+pub static MZ_PREPARED_STATEMENT_HISTORY_REDACTED: Lazy<BuiltinView> = Lazy::new(|| BuiltinView {
     name: "mz_prepared_statement_history_redacted",
     schema: MZ_INTERNAL_SCHEMA,
     column_defs: None,
     // everything but "sql"
     sql: "
 SELECT id, session_id, name, redacted_sql, prepared_at, statement_type
 FROM mz_internal.mz_prepared_statement_history",
-    sensitivity: DataSensitivity::SuperuserAndSupport,
-};
+    access: vec![SUPPORT_SELECT, MONITOR_REDACTED, MONITOR],
+});
 
 pub static MZ_SESSION_HISTORY: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
     name: "mz_session_history",
     schema: MZ_INTERNAL_SCHEMA,
     data_source: Some(IntrospectionType::SessionHistory),
     desc: MZ_SESSION_HISTORY_DESC.clone(),
     is_retained_metrics_object: false,
-    sensitivity: DataSensitivity::SuperuserAndSupport,
+    access: vec![SUPPORT_SELECT],
```

Comment:
Seems like this is missing a few roles?
```suggestion
    access: vec![SUPPORT_SELECT, MONITOR_REDACTED, MONITOR],
```

Reply:
we are going to make this public, which is why I didn't bother changing it. Will do that in a separate PR.